### ISSUE TITLE
Highlighted text in light mode

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#352]
When using keyboard navigation and light mode, highlighting items makes the text impossible to read. It is a dark blue highlight over the black text, which is impossible to read. The text should change to white so it is readable.

Locations:
- Table filter chips
- Navigation bar entries
- Admin Systems namespace select options
